### PR TITLE
[BC-18080] Add methodologies for Mobile iOS

### PR DESCRIPTION
**Pull request by bharat619**
Jira: [BC-18080](https://bugcrowd.atlassian.net/browse/BC-18080)

Methodology Document: [Mobile iOS](https://docs.google.com/document/d/1SMhe2igtpldit5SuGrqLT-MBpYymtHIp9UyktJ1Thtg/edit#)

To Test:
Content must be same as in the methodology document

### CODE REVIEW

**Review by arcwhite — DISMISSED** [review]

File: methodologies/mobile_ios.json [hunk 1]
```diff
@@ -0,0 +1,440 @@
+{
+  "metadata": {
+    "title": "Binaries",
```
Comment (arcwhite) [inline methodologies/mobile_ios.json:3]: Binaries!? I don't think this is right

Comment (bharat619) [reply methodologies/mobile_ios.json]: Wopsies. Sorry about that. Copy paste blunder. 

I did double checked for any other mistake like this now.

**Review by arcwhite — APPROVED** [review]

**Comment by fedetaglia** [comment]
@bharat619 ooo my bad, if we want these new methodologies to have the "upload log" and the "executive summary" step we need to manually add them to the methodology

```
      {
        "key": "upload_logs",
        "title": "Upload logs",
        "description": "This should include all associated traffic associated to the in-scope targets.",
        "type": "large_upload"
      },
      {
        "key": "executive_summary",
        "title": "Executive summary",
        "description": "The executive summary should be written with a high-level view of both risk and business impact. It should be concise and clear, therefore it is important to use plain English. This ensures that non-technical readers can gain insight into security concerns outlined in your report.",
        "type": "executive_summary"
      }
```

see the `website` methodology https://github.com/bugcrowd/methodology-taxonomy/blob/main/methodologies/website_testing.json#L872

can you pls double-check this and the others?

**Comment by bharat619** [comment]
@fedetaglia Raised a PR for it https://github.com/bugcrowd/methodology-taxonomy/pull/12

**Pull request merged**